www/docs/install.md

```diff
@@ -142,9 +142,8 @@ You can verify it using [our public key](https://goreleaser.com/static/gorelease
       ```
     1. Verify the signature:
       ```sh
-      cosign verify-blob \
-        -key https://goreleaser.com/static/goreleaser.pub \
-        -signature checksums.txt.sig \
+      COSIGN_EXPERIMENTAL=1 cosign verify-blob \
+        --signature checksums.txt.sig \
```

Comment:
once we enable keyless mode in GoReleaser, people can use our certificate and signature while verifying without requiring experimental mode. So, the command will be like this:

```shell
$ cosign verify-blob --signature checksums.txt.sig --cert checksums.pem checksums.txt
Verified OK
```

![Screen Shot 2021-12-09 at 20 27 07](https://user-images.githubusercontent.com/16693043/145446076-b26d0981-2585-4144-96f8-1a3f5cec00b6.png)

WDYT @caarlos0 @dirien @dentrax?